src/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <title>Registration form</title>
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+  <body>
+
+    <form action="" name="registration" id="registrationForm">
+      <label for="fullName">Write your full name, please:</label>
+      <input type="text" name="fullname" id="fullName" placeholder="Full name" autocomplete="off">
+     
+      <div class="hidden-info">
+        <input type="checkbox" id="hidden-info" name="hidden-info" value="hidden-info">
+        <label for="hidden-info">I would prefer not to specify my phone number and home town</label>
+      </div>
+
+      <div id="privateInfo">
+        <label for="phoneNumber">Enter your phone number, please:</label>
+        <input type="tel" name="phone" id="phoneNumber" placeholder="Phone number" autocomplete="off">
+        
+        <label for="region">Select your region:</label>
+        <select name="region" id="region">
+          <option value="" selected disabled></option>
+          <option value="Kyiv">Kyiv</option>
+          <option value="Center">Center</option>
+          <option value="North">North</option>
+          <option value="East">East</option>
+          <option value="South">South</option>
+          <option value="West">West</option>
+        </select>
+
+        <div id="cities" class="display-none">
+        <label for="city">Select your city:</label>
+        <select name="city" id="city">
+          <option value="" selected disabled></option>
+          <option value="Cherkasy" data-region="center" class="display-none">Cherkasy</option>
```

Comment:
you might want to store the structure of the regions in js code and render the select dynamically.